source/includes/aggregation/atlas-search.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
```

Comment:
The `use` statements are missing in all files.

```suggestion

use MongoDB\Builder\Pipeline;
use MongoDB\Builder\Search;
use MongoDB\Builder\Stage;
```
```

Reply:
I haven't been able to test this code as its in an unreleased version, but I'll go back and fix up the code files so they are as runnable as possible

Reply:
In `composer.json`, you can add allow the installation of a dev version by setting `
    "mongodb/mongodb": "^1.20@dev"`